### PR TITLE
XIVY-13641 Add Form Editor webview editor 

### PR DESCRIPTION
**Pull request by ivy-lli**
Based on core PR: https://github.com/axonivy/core/pull/5329

### CODE REVIEW

File: extension/src/form-editor/form-editor-provider.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+import { findRootEntry, parseBuildManifest } from '../base/build-manifest';
+import { calcNonce } from '../base/webview-nonce';
+import { Messenger } from 'vscode-messenger';
+import { FormWebSocketMessage, WebSocketForwarder } from '../websocket-forwarder';
+import { NotificationType } from 'vscode-messenger-common';
+
+const WebviewReadyNotification: NotificationType<void> = { method: 'ready' };
+const InitializeConnectionRequest: NotificationType<{ file: string }> = { method: 'initializeConnection' };
+
+export default class FormEditorProvider implements vscode.CustomTextEditorProvider {
```
Comment (ivy-lmu) [inline extension/src/form-editor/form-editor-provider.ts:11]: so it is vscode.CustomTextEditorProvider

**Review by ivy-lmu — APPROVED** [review]
cool 👍

File: extension/src/form-editor/form-editor-provider.ts (hunk 1 above)
Comment (ivy-lli) [reply extension/src/form-editor/form-editor-provider.ts:11]: ah copy paste from the variable editor... hm... need to check that

Comment (ivy-lli) [reply extension/src/form-editor/form-editor-provider.ts:11]: will check it, when I check the dirty state of the editor...

**Pull request merged**